<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sendTwitDialog</class>
 <widget class="QDialog" name="sendTwitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>101</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>445</width>
    <height>89</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>511</width>
    <height>101</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Send Tweet</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/images.qrc">
    <normaloff>:/twitter/im-status-message-edit.svg</normaloff>:/twitter/im-status-message-edit.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;B&gt;Type Your Twit Here&lt;/B&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>twitLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="LineEdit" name="twitLineEdit">
     <property name="minimumSize">
      <size>
       <width>361</width>
       <height>20</height>
      </size>
     </property>
     <property name="maxLength">
      <number>140</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="charsLeft">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>121</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string>No. of characters entered</string>
     </property>
     <property name="whatsThis">
      <string>This shows the number of characters entered so far out of the total permissible limit for a tweet</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>000 of 140</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>85</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Send Tweet</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>112</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>I changed my mind!!!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">lineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>twitLineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>sendTwitDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>sendTwitDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
